pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=63.0"]
+requires = ["setuptools>=63.0", "unasync"]
```

Comment:
We shouldn't need unasync to build a wheel, right?

Reply:
Now that we're using pre-commit for the synchro script, yup! Good catch.